src/main/statistics_gathering/solution_graph_statistics.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2020 Anastasiia Birillo, Elena Lyulina
+
+from collections import defaultdict
+from typing import Tuple, Dict, Union, List
+
+from src.main.canonicalization.consts import TREE_TYPE
+from src.main.solution_space.solution_graph import SolutionGraph
+from src.main.canonicalization.ast_tools import get_vertices_number_in_ast
+
+
+def get_general_solution_graph_statistics(solution_graph: SolutionGraph) -> Tuple[Dict[int, int], Dict[int, int]]:
```

Comment:
Again, let's be more specific about statistics. We can have a different statistic for our graph in the future, so we should always describe it in detail.